src/python/palindromos.py

```diff
@@ -0,0 +1,28 @@
+# 1 - open your terminal and type: pip install pythonds
+
+# 2 - of this package below, download the 'Deque' module 
+from pythonds.basic.deque import Deque
```

Comment:
Para evitar do usuário ter que instalar pacotes externos, poderia usar o módulo built-in `collections.deque`: https://docs.python.org/3/library/collections.html#collections.deque

De qualquer maneira, é possível resolver o problema de maneira mais simples, sem a necessidade de usar deque.